STEPS.md

```
@@ -2848,6 +2849,142 @@ Given the following menu links exist in the menu "Main navigation":
 >  - Assert presence and content of meta tags with proper attribute handling.
 
 
+## Drupal\ModuleTrait
+
+[Source](src/Drupal/ModuleTrait.php), [Example](tests/behat/features/drupal_module.feature)
+
+>  Enable and disable Drupal modules with automatic state restoration.
```

markdownlint-cli2 (0.18.1)

Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)